app/add-details/page.tsx

```diff
@@ -170,9 +184,9 @@ export default function Home() {
               plant={plantsToAdd[currentIndex - 1]}
               date={details[currentIndex - 1].date_added ?? getDefaultDate()}
               plantingType={details[currentIndex - 1].planting_type ?? ''}
-              onDateChange={date =>
-                updateInput('date_added', date, currentIndex - 1)
-              }
+              onDateChange={date => {
```

Comment:
Kinda a nit but you can take out the extra curly brace around this function just for consistency with the other props.